js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,15 @@
+function animalSort(arr) {
+  const sortedByLegs = arr.sort(function (animalA, animalB) {
+    if (animalA.numberOfLegs !== animalB.numberOfLegs) {
+      return animalA.numberOfLegs - animalB.numberOfLegs;
+    } else if (animalA.numberOfLegs === animalB.numberOfLegs) {
+      let aAnimalName = animalA.name.toLowerCase();
+      let bAnimalName = animalB.name.toLowerCase();
+      return aAnimalName < bAnimalName ? -1 : aAnimalName > bAnimalName ? 1 : 0;
+    }
+  });
+
+  return sortedByLegs;
+}
+
+export { animalSort };
```

Comment:
This question shouldn't be a part of this PR

Reply:
Have deleted in new commit